src/Select.tsx

```diff
@@ -92,6 +92,7 @@ export function Select(props: SelectProps) {
 						value={value}
 						onChange={onChange}
 						multiple={multiple}
+						label={!!label ? label : undefined}
```

Comment:
`!!label && undefined`

Reply:
Uh.. `Select.test.tsx` passes label prop as `label={label && 'Test'}` which evaluates to `'Test'` for truthy and `false` when falsy. Hence, the explicit undefined. IMO this should be simplified as:

src/Select.tsx
```
label={label}
```

test/Select.test.tsx
```
label={label ? label : undefined}
```